api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_highly_variable_genes.py

```diff
@@ -185,9 +190,11 @@ def _highly_variable_genes_seurat_v3(
                 acc.update(var_dim, data)
 
         counts_sum, squared_counts_sum = acc.finalize()
-        norm_gene_vars = (1 / ((n_samples - 1) * np.square(reg_std.T))).T * (
-            (n_samples * np.square(batches_u.T)).T + squared_counts_sum - 2 * counts_sum * batches_u
-        )
+        with np.errstate(divide="ignore", invalid="ignore"):
```

Comment:
I don't understand the implication of ignoring these errors, so maybe worth a comment?

Reply:
> I don't understand the implication of ignoring these errors, so maybe worth a comment?

It is quite literally what the numpy docs say - we want the IEEE-specified value for these cases, not the Python exception. E.g., we want 1/0 to be inf, and 0/0 to be NaN, etc. Is that the kind of info you are looking for, or a deeper explanation of the algo, or?